PocketKit/Sources/PocketKit/Article/ReadableViewModel/RecommendationViewModel.swift

```diff
@@ -80,12 +80,8 @@ class RecommendationViewModel: ReadableViewModel {
         recommendation.item?.bestURL
     }
 
-    func moveToSaves() {
-        guard let savedItem = recommendation.item?.savedItem else {
-            return
-        }
-
-        source.unarchive(item: savedItem)
+    var isArchived: Bool {
+        return recommendation.item?.savedItem?.isArchived ?? false
```

Comment:
This line reaches very deeply into recommendation.
Can we add a function to recommendation that returns if it's saved item is archived instead?
Otherwise we are tightly coupling the ViewModel to it.

Reply:
discussed over slack, as of now, will keep this change as we're currently doing this in multiple places. If we want to refactor this, we can create a separate ticket to rethink our architecture in general when reaching for the `isArchived` property for our view models in general